webpack.config.prod.babel.js

```diff
@@ -75,6 +75,18 @@ module.exports = {
       },
     }),
 
+    // Only load syntax highlighting for Python
+    new webpack.ContextReplacementPlugin(
+      /highlight\.js\/lib\/languages$/,
+      new RegExp('^./(python)$')
+    ),
+
+    // Only load locale for Fr and En
+    new webpack.ContextReplacementPlugin(
+      /react-intl\/locale-data$/,
+      new RegExp('^./(fr|en)$')
```

Comment:
Set language according to configuration?

Reply:
Done!